types/ember__application/test/owner.ts

```diff
@@ -0,0 +1,11 @@
+import Owner from '@ember/owner';
+import { getOwner, setOwner } from '@ember/application';
+import EmberObject from '@ember/object';
+
+declare let owner: Owner;
+
+class MyClass extends EmberObject {}
+const classInstance = new MyClass();
+
+setOwner(classInstance, owner);
+getOwner({}); // $ExpectType Owner
```

Comment:
This here is actually exactly why the above comment matters: the actual answer here at runtime would be `undefined`, and people would have a very bad time with "undefined is not a function" as a result. :sob:

```suggestion
getOwner({}); // $ExpectType Owner | undefined
```